airflow/tests/k8s_sample/dags/task_fail.py

```diff
@@ -1,4 +1,4 @@
-# (C) Datadog, Inc. 2020-present
```

Comment:
Could you revert this?

Reply:
The pipeline failed for license header validation, so I updated it via ddev. Should this not be updated?

Reply:
ah I see. The command compares with master and not the branch you target: 

https://github.com/DataDog/integrations-core/blob/b7c15a6882af399c6f477b7c751c7b0dac23a0ce/datadog_checks_dev/datadog_checks/dev/tooling/license_headers.py#L29-L37

Could you create a card? Also, I would not change it to avoid messing up with a potential future release since we know it's a bug in the validation command

Reply:
Created [AITOOLS-238](https://datadoghq.atlassian.net/browse/AITOOLS-238). I see both these files are already removed in [master](https://github.com/DataDog/integrations-core/pull/15502/files#diff-1c96f1e740f9820754b42f731f1452dc757614bf4b0bbbc838b87cb754765f76), so I don't see a potential problem with future release.

Any concerns to leave it as such for this PR?

Reply:
Cool, will wait for the review approval before merging :)

Reply:
I'll let @iliakur have a lot as the RM for 7.47.x 🙂